helm/prometheus-rules/templates/_helpers.tpl

```diff
@@ -34,3 +34,11 @@ rocket
 phoenix
 {{- end -}}
 {{- end -}}
+
+{{- define "hasToBeCancelled" -}}
```

Comment:
I would probably rename this field to ensure this describe things best. Why are you cancelling alert outside of working hours ?

Reply:
because CAPO is not yet prod ready

Reply:
Yes that's what I was thimking :D Could we call that beta or testing instead ?

Reply:
isBeta for example ?

Reply:
but my thinking it could have other reasons why we could inhibit them. @kopiczko what do you think? you are good at naming

Reply:
Makes sense yes :)

Reply:
> @kopiczko what do you think?

I don't mind. If this is the only reason for this thing to exist then probably changing the name makes sense.